lib/go-tc/deliveryservices.go

```diff
@@ -428,8 +445,8 @@ func (ds *DeliveryServiceNullable) validateTypeFields(tx *sql.Tx) error {
 					if *ds.RangeRequestHandling == 3 {
 						return validation.Validate(ds.RangeSliceBlockSize, validation.Required,
 							// Per Slice Plugin implementation
-							validation.Min(262144),   // 256KiB
-							validation.Max(33554432), // 32MiB
+							validation.Min(MinRangeSliceBlockSize), // 256KiB
```

Comment:
We should do some basic validation for `MaxRequestHeaderBytes` as well -- like it should be greater than zero.